Packs/AWS-Organizations/Integrations/AWSOrganizations/AWSOrganizations.yml

```diff
@@ -693,5 +693,6 @@ fromversion: 6.10.0
 marketplaces:
 - xsoar
 - marketplacev2
+- xpanse
```

Comment:
Make sure to document the changes in a release note and to update the dockerimage.

Reply:
[AWS org RN](https://github.com/demisto/content/pull/31951/commits/5e52279fbe72f65a2b134d7bdf904ba46269d3b7)